scripts/globals/mobskills/Gigaflare.lua

```diff
@@ -0,0 +1,35 @@
+---------------------------------------------
+--  Gigaflare
+--  Family: Bahamut
+--  Description: Deals massive Fire damage to enemies within a fan-shaped area. 
+--  Type: Magical
+--  Utsusemi/Blink absorb: Wipes shadows
+--  Range: 
+--  Notes: Used by Bahamut when at 10% of its HP, and can use anytime afterwards at will.
+---------------------------------------------
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+---------------------------------------------
+
+function onMobSkillCheck(target,mob,skill)
+    local result = 1;
+    local mobhp = mob:getHPP();
+
+    if (mobhp <= 10) then
+        result = 0;
+    end;
+
+    return result;
+end;
+
+function onMobWeaponSkill(target, mob, skill)
+    local gigaflareCount = mob:getLocalVar("GigaFlare") + 1; -- increments the variable for scripted use of Gigaflare
```

Comment:
still wouldn't need incremented again. would need to not increment to 1 form zero till it actually foes off, for that scenario.

>